lib/include/hse/ikvdb/kvdb_rparams.h

```diff
@@ -96,7 +96,7 @@ struct kvdb_rparams {
     uint32_t c0_ingest_threads;
     uint16_t cn_maint_threads;
     uint16_t cn_io_threads;
-    uint32_t cndb_compact_hwm_pct;
+    double cndb_compact_hwm_pct;
```

Comment:
Might want to double check padding if something needs to be done.

Reply:
Other than optimizing structure size we shouldn't have to worry about filling holes.  Is there a problem if we don't?

Reply:
Nah. I just know in the past we have cared about it at times. This seems like a time we can not worry about it.